### DISCUSSION TITLE
can not start Swar

### DISCUSSION THREAD

**Discussion by vuonganvn** [swar/Swar-Chia-Plot-Manager#1038] [Q&A]
(venv) C:\Users\vuong\Downloads\Swar-Chia-Plot-Manager-main>python manager.py start
Traceback (most recent call last):
  File "C:\Users\vuong\Downloads\Swar-Chia-Plot-Manager-main\manager.py", line 30, in <module>
    start_manager()
  File "C:\Users\vuong\Downloads\Swar-Chia-Plot-Manager-main\plotmanager\library\utilities\commands.py", line 23, in start_manager
    raise ManagerError('Manager is already running.')
plotmanager.library.utilities.exceptions.ManagerError: Manager is already running.

swar was displayed when I tried with the command: python manager.py start
please help me
thank very much

**Comment by Chilli71** [comment]
If it pretends it's running, did you try to stop or restart it? (python manager.py restart or stop)?
Please be arare that stopping the maganger does not stop the plots it kicked off.
What does it show when you try to view it (python manager.py view)? If you start view and it is not running it writes manager is stopped.

**Reply by vuonganvn** [reply]
thank you for reply, i looked with python3 manager.py view command, actually it still runs
